source/xep/stanzaId/StanzaId.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
```

Comment:
Make one file for each class. Separate the header from implementation.

Reply:
Done, I wasn't that fond of having one header, but Swiften structure makes me declare a few relatively trivial classes and I was not sure about having 8 files in this subdirectory for a relatively minor feature...